Microsoft.DotNet.Interactive/ISupportNuget.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Interactive.Utility;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public interface ISupportNuget
+    {
+        public abstract void AddScriptReferences(IReadOnlyList<ResolvedPackageReference> packageReferences);
+
```

Comment:
Are these script references or package references?

A `ResolvedPackageReference` should only be added by the implementation. I'm not sure it belongs on the interface.

Reply:
What do you suggest?

Reply:
I'm trying to understand the use case. 

We currently differentiate between a `PackageReference` type and a `ResolvedPackageReference`. Neither of them seems appropriate for a "script reference".